Documentation/CONFIGURATION.md

```diff
@@ -0,0 +1,161 @@
+# Configuring the Fastly_Cdn Module
+
+This document describes how to configure the Fastly_Cdn module and connect it
+to Fastly's CDN services.
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Configure the Fastly Service](#configure-the-fastly-service)
+* [Basic Configuration](#configure-the-module)
+* [Further Configuration Options](#further-configuration-options)
+
+## Prerequisites
+
+To proceed with configuration you will need to have
+[signed up for Fastly's service](https://www.fastly.com/signup). You will need
+the API key (or a user key with purge access). Details of how to find these
```

Comment:
There's no UI for tokens yet, so I'd like to keep this simple. That can be changed and recommended when there's a UI to help.